portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,11 +25,23 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> facts;
+
+  @Override
+  public void init() {
+      facts = new ArrayList<>();
+
+      facts.add("I'm one of a triplet - and we are all females");
+      facts.add("My big brothers are twins");
+      facts.add("The names of my entire family start with 'א'");
+      facts.add("I have played handball for 8 years, 13 games seasons");
+ }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<link rel=\"stylesheet\" href=\"style.css\">");
-    response.getWriter().println("<h1>Hello Orpaz!</h1>");
+        String fact = facts.get((int) (Math.random() * facts.size()));
+
+    response.setContentType("text/html; charset=UTF-8");
+    response.getWriter().println(fact);
```

Comment:
Are you sure this is correct? I believe it will write out the fact without any HTML markup at all, which isn't really conformant.
You may wish to add some more text around here - e.g., leave in the stylesheet, and output "h1" and "/h1" around the fact text.

Reply:
(Actually, looking at the way you use this - looks like you *are* correct, but the content-type may need to be just text/plain instead.)